attributes/jenkins.rb

```diff
@@ -0,0 +1,40 @@
+default['jenkins']['executor']['timeout'] = 240
+
+# Confirm plugins node attribute array exists
+default['jenkins_ci']['jenkins']['plugins'] = [] if default['jenkins_ci']['jenkins']['plugins'].nil
```

Comment:
The examples I found for creating an attribute array if one did not exist used this format.